src/Reflection/Adapter/ReflectionClass.php

```diff
@@ -33,6 +33,12 @@
  */
 final class ReflectionClass extends CoreReflectionClass
 {
+    /** @internal */
+    public const SKIP_INITIALIZATION_ON_SERIALIZE_COMPATIBILITY = 8;
+
+    /** @internal */
+    public const SKIP_DESTRUCTOR_COMPATIBILITY = 16;
```

Comment:
I wonder if we should `@inheritDoc` on these, so we understand they come from above